src/WebJobs.Script.WebHost/WebHostServiceCollectionExtensions.cs

```diff
@@ -40,6 +43,8 @@
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
+using Polly;
```

Comment:
The newly added usings (aside from where you've added the extension method) are no longer required

Reply:
Thanks removed.